protostar.py

```diff
@@ -3,11 +3,11 @@
 from argparse import ArgumentParser
 from pathlib import Path
 from re import Pattern
-
 import argparse
-
 from src import cli
 
+SCRIPT_ROOT = Path(__file__).parent
```

Comment:
can't we just import the script root later on, instead of passing it as a parameter with the calls?